server/README.md

```diff
@@ -0,0 +1,73 @@
+## Request & Response Examples
+The server provides a simple layer for managing agent processes.
+
+### API Resources
+
+  - [POST /start](#get-magazines)
+  - [POST /stop](#get-magazinesid)
+  - [POST /ping](#post-magazinesidarticles)
+
+
+### POST /start
+This api starts an agent with given graph and override properties. The started agent will join into the specified channel, and subscribe to the uid which your browser/device's rtc use to join.
+
+| Param    | Description |
+| -------- | ------- |
+| request_id  | any uuid for tracing purpose    |
+| channel_name | channel name, it needs to be the same with the one your browser/device joins, agent needs to stay with your browser/device in the same channel to communicate  |
+| chat_uid    | the uid which your browser/device's rtc use to join, agent needs to know your rtc uid to subscribe your audio    |
```

Comment:
Is `chat_uid` a good name here? maybe `uid` is good enough?

Reply:
uid is bit ambiguous as it can refer to bot uid.. or maybe we should expose both so that it's clearer?

Reply:
few options,
1. human uid
2. adding optional bot_uid, so it's clearer when using

Reply:
Agree that expose both `uid` and `bot_uid`(or maybe `agent_uid`?) seems better.

Reply:
end up with user_uid and bot_uid